### PR TITLE
 feat(attendee report): Add tooltips for attendee report.

### PR DESCRIPTION
**Pull request by Camwyn**
Add functions for attendee report tooltips and use them in `Attendance_Totals.php`.

🎫 https://central.tri.be/issues/120856

Required by: https://github.com/moderntribe/event-tickets-plus/pull/768 and https://github.com/moderntribe/event-tickets-plus/pull/772
Requires: https://github.com/moderntribe/tribe-common/pull/978 (styles)

### CODE REVIEW

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 1]
```diff
@@ -50,4 +50,44 @@ public function integrate_with_attendee_screen() {
...
+
+	/**
+	 * Get Attendee Total Sold Tooltip
+	 *
+	 * @since TBD
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_sold_tooltip() {
+		ob_start();
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]: Should we move this towards an `admin-view` file?

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 2]
```diff
@@ -50,4 +50,44 @@ public function integrate_with_attendee_screen() {
...
+	}
+
+	/**
+	 * Get Attendee Total Completed Orders Tooltip
+	 *
+	 * @since TBD
+	 *
+	 * @return string a string of html for the tooltip
+	 */
+	public function get_total_conpleted_tooltip() {
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]: Should we move this towards an `admin-view` file?

File: src/Tribe/Commerce/PayPal/Attendance_Totals.php [hunk 3]
```diff
@@ -70,20 +70,20 @@ protected function should_count( Tribe__Tickets__Ticket_Object $ticket ) {
...
-		$total_paid      = $this->get_total_complete();
+		$total_complete  = $this->get_total_complete();
 		$total_cancelled = $this->get_total_cancelled();
 
-		echo "
-			<ul>
-				<li> <strong>$total_sold_label&nbsp;</strong>$total_sold </li>
-				<li> $total_paid_label&nbsp;$total_paid </li>
-				<li> $total_cancelled_label $total_cancelled </li>
+		?><ul>
```
Comment (bordoni) [inline src/Tribe/Commerce/PayPal/Attendance_Totals.php]: Should we move this towards an `admin-view` file?

File: src/Tribe/Abstract_Attendance_Totals.php (hunk 1 above)
Comment (jesseeproductions) [reply src/Tribe/Abstract_Attendance_Totals.php]: I would like to us continue with the tooltips being in similar setup to the status manager ones on the Order page: https://github.com/moderntribe/event-tickets/blob/master/src/Tribe/Status/Abstract_Commerce.php#L163

File: src/Tribe/Abstract_Attendance_Totals.php (hunk 2 above)
Comment (Camwyn) [reply src/Tribe/Abstract_Attendance_Totals.php]: also, typo 🤦‍♂️

**Comment by Camwyn** [comment]
@sc0ttkclark @bordoni @jesseeproductions To respond and continue the discussion on moving this somewhere:
I have created a factory method (that is restricted to `src/Tribe/Abstract_Attendance_Totals.php` for now) as a test of what we'd like to see.

I know Brian has been working with these extensively, so value his input on this **highly** ( some of the newer tooltips are much more complex, so this might just be a thing for the "easy" ones 😉 )

This should probably live in `common` but TBH I am not sure the best place for it, so I am desiring more input on that as well.

**Comment by sc0ttkclark** [comment]
I think this would be useful in common, but I'll get @bordoni's point of view on this during the lead meeting today.

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 4]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+			'context'   => '',
+		];
+
+		$merged_args = wp_parse_args( $args, $default_args );
+
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
+			<span class="dashicons dashicons-<?php esc_attr_e( $merged_args[ 'icon' ] ); ?> <?php esc_attr_e( $merged_args[ 'additional_classes' ] ); ?>"></span>
+			<div class="<?php esc_attr_e( $merged_args[ 'direction' ] ); ?>">
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
			<div class="<?php echo sanitize_html_class( $merged_args[ 'direction' ] ); ?>">
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 5]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+				<?php if ( is_array( $message ) ) {
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
				<?php if ( is_array( $message ) ) :
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 6]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+					foreach( $message as $mess ) { ?>
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
					foreach( $message as $mess ) : ?>
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 7]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+						<p>
+							<span><?php echo wp_kses_post( $mess ); ?><i></i></span>
+						</p>
+					<?php }
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
					<?php endforeach;
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 8]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+				} else { ?>
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
				else : ?>
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 9]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+					<p>
+						<span><?php echo wp_kses_post( $message ); ?><i></i></span>
+					</p>
+				<?php } ?>
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
				<?php endif; ?>
```

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 10]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+			</div>
+		</div>
+		<?php
+		$html = ob_get_clean();
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]: Can we move this into `admin-views`?

File: src/Tribe/Abstract_Attendance_Totals.php [hunk 11]
```diff
@@ -50,4 +50,88 @@ public function integrate_with_attendee_screen() {
...
+			'direction' => 'down',
+			'context'   => '',
+		];
+
+		$merged_args = wp_parse_args( $args, $default_args );
+
+		ob_start();
+		?>
+		<div class="tribe-tooltip" aria-expanded="false">
+			<span class="dashicons dashicons-<?php esc_attr_e( $merged_args[ 'icon' ] ); ?> <?php esc_attr_e( $merged_args[ 'additional_classes' ] ); ?>"></span>
```
Comment (bordoni) [inline src/Tribe/Abstract_Attendance_Totals.php]:
```suggestion
			<span class="dashicons dashicons-<?php echo sanitize_html_class( $merged_args[ 'icon' ] ); ?> <?php echo sanitize_html_class( $merged_args[ 'additional_classes' ] ); ?>"></span>
```

File: src/Tribe/Commerce/PayPal/Attendance_Totals.php (hunk 3 above)
Comment (sc0ttkclark) [reply src/Tribe/Commerce/PayPal/Attendance_Totals.php]: I guess we could here, it's small but might as well stick to precedence

**Pull request merged**